generate-training-data.py

```diff
@@ -1,13 +1,32 @@
 import os
 import cv2
+cap = cv2.VideoCapture(0)
+path = 'path_to_save_images'
+start = False
+count = 0
+while True:
+    ret, frame = cap.read()
+    if not ret:
+        continue
+    if count == 300:
+        break
 
-# write your code here to open your laptop's camera
-# and store images for rock paper scissors
+    cv2.rectangle(frame, (100, 100), (508, 580), (0, 0, 255), 2)
 
-# I would highly recommend storing these images in folders stucture like below:
-# training_data\
-#           |-- empty\
-#           |-- rock\
-#           |-- paper\
-#           |-- scissors\
-# having a folder structure like this will make it easy for you to read the images while pre-processing
+    if start:
+        roi = frame[100:500, 100:500]
+        save_path = os.path.join(path, '{}.jpg'.format(count + 1))
```

Comment:
Try following the folder structured suggested, create folders where necessary and don't expect the user to come and  change your code like in:
```
path = 'path_to_save_images'
```